DirectProgramming/C++SYCL_FPGA/Tutorials/Features/task_sequences/README.md

```diff
@@ -0,0 +1,250 @@
+
+## Sample Structure
+The 2 different example designs in this sample perform similar operations.
+
+1. [Naive](naive/) This implementation uses arrays to pass data between loops and each loop are scheduled to run sequentially.
+2. [Task sequence](task_sequence/) This implementation uses inter-kernel pipes to pass data between loops and applies task sequences to schedule loops to run concurrently.
+
+## Build a Design
+
+All designs in this sample supports four compilation options: Emulator, Simulator, Optimization Report, FPGA Hardware. For details on the different compilation options, see the [fpga_compile](https://github.com/oneapi-src/oneAPI-samples/tree/master/DirectProgramming/C%2B%2BSYCL_FPGA/Tutorials/GettingStarted/fpga_compile) tutorial.
+
+Use the appropriate TYPE parameter when running CMake to config which design to compile:
+| Example                                      | Directory             | Type (-DTYPE=) |
+|----------------------------------------------|-----------------------|----------------|
+| Naive                                        | naive/                | `NAIVE`        |
+| Task sequence                                | task-sequence/        | `TASK_SEQUENCE`|
+
+>**Note**: When working with the command-line interface (CLI), you should configure the oneAPI toolkits using environment variables. Set up your CLI environment by sourcing the `setvars` script in the root of your oneAPI installation every time you open a new terminal window. This practice ensures that your compiler, libraries, and tools are ready for development.
+>
+> Linux*:
+> - For system wide installations: `. /opt/intel/oneapi/setvars.sh`
+> - For private installations: ` . ~/intel/oneapi/setvars.sh`
+> - For non-POSIX shells, like csh, use the following command: `bash -c 'source <install-dir>/setvars.sh ; exec csh'`
+>
+> Windows*:
+> - `C:\Program Files(x86)\Intel\oneAPI\setvars.bat`
+> - Windows PowerShell*, use the following command: `cmd.exe "/K" '"C:\Program Files (x86)\Intel\oneAPI\setvars.bat" && powershell'`
+>
+> For more information on configuring environment variables, see [Use the setvars Script with Linux* or macOS*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-linux-or-macos.html) or [Use the setvars Script with Windows*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-windows.html).
+
+### On Linux*
+
+1. Change to the sample directory.
+2. Build the program for Intel® Agilex® 7 device family, which is the default.
+   ```
+   mkdir build
+   cd build
+   cmake .. -DTYPE=<NAIVE/TASK_SEQUENCE>
+   ```
+   > **Note**: You can change the default target by using the command:
+   >  ```
+   >  cmake .. -DFPGA_DEVICE=<FPGA device family or FPGA part number> -DTYPE=<NAIVE/TASK_SEQUENCE>
```

Comment:
Isn't this sample supported on BSPs?
If so please use this template:
https://github.com/oneapi-src/oneAPI-samples/blob/development/DirectProgramming/C%2B%2BSYCL_FPGA/Tutorials/Features/ac_int/README.md?plain=1#L161-L175

Same thing for instructions n Windows

Reply:
This sample will only work with a BSP that supports host pipes. I didn't think our customers had access to such a BSP.